docs/standard/choosing-core-framework-server.md

```diff
@@ -102,6 +102,6 @@ Some Microsoft or third-party platforms don’t support .NET Core. Some Azure se
 - [Target frameworks](frameworks.md)
 - [.NET Core Guide](../core/index.md)
 - [Porting from .NET Framework to .NET Core](../core/porting/index.md)
-- [.NET Framework on Docker Guide](../framework/docker/index.md)
+- [Introduction to .NET and Docker](../core/docker/index.md)
```

Comment:
The index files are disappearing, so the appropriate link is
```suggestion
- [Introduction to .NET and Docker](../core/docker/intro-net-docker.md)
```

Reply:
I've made the suggested change.